.appveyor.yml

```diff
@@ -65,6 +65,7 @@ install:
 
   # Install requirements from inside conda environment
   - cmd: activate testenv
+  - cmd: conda install libpython m2w64-toolchain -c msys2
```

Comment:
Could we add some comment here and below in for azure-pipelines why we need this for others to understand?